src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,100 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+const props = {
+  tabs: {
+    selected: 0,
+    triggerHref: '#',
+    role: 'navigation',
+  },
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
```

Comment:
You're missing this div wrapper: `<div className="bx--grid bx--grid--full-width landing-page">`

Reply:
Instead of replacing the landing-page-banner div, you should have added this a new wrapper. Please try again and be sure to review your changes locally before pushing them to this PR for review.

Reply:
You actually need both of these div wrappers: 
 <div className="bx--grid bx--grid--full-width landing-page">
    <div className="bx--row landing-page__banner">

Please try again and be sure to review your changes locally before pushing them to this PR for review.